pkg/kube/translation/ingress.go

```diff
@@ -29,16 +29,23 @@ import (
 
 	"github.com/apache/apisix-ingress-controller/pkg/id"
 	apisixv12 "github.com/apache/apisix-ingress-controller/pkg/kube/apisix/apis/config/v1"
+	configv2alpha1 "github.com/apache/apisix-ingress-controller/pkg/kube/apisix/apis/config/v2alpha1"
```

Comment:
Please merge master， the `v2alpha1` no longer exist, use `v2beta3` instead.

Reply:
a lot of changes were made.
I've done the changes